File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples41.vb
' <snippetlinqexamples41>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim query_pagingsort1 = ( _
      From c In svcContext.ContactSet _
      Where c.LastName IsNot "Parker" _
      Order By c.LastName Ascending, c.FirstName Descending _
      Select New With {Key c.FirstName,
                       Key c.LastName}).Skip(2).Take(2)
  For Each c In query_pagingsort1
   Console.WriteLine(c.FirstName &amp; " " &amp; c.LastName)
  Next c
 End Using

' </snippetlinqexamples41>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/linqexamples41.cs
// <snippetlinqexamples41>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  var query_pagingsort1 = (from c in svcContext.ContactSet
                           where c.LastName != "Parker"
                           orderby c.LastName ascending,
                           c.FirstName descending
                           select new { c.FirstName, c.LastName })
                           .Skip(2).Take(2);
  foreach (var c in query_pagingsort1)
  {
   System.Console.WriteLine(c.FirstName + " " + c.LastName);
  }
 }

// </snippetlinqexamples41>